### PR TITLE
API Augment update to use metadata v15

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to update API Augment to use the newer metadata v15 provided with Polkadotjs/api v 14.1.1

Part of #2154 

# Discussion

- Updated the Export Metadata Cmd to use Metadata v15
- Update the fetch:local to use Metadata v15
- Fixed a test that doesn't work under all situations (description is not always returned on all networks)
- Setup e2e testnet workflow to run on the provided branch instead of against the release branch so that other test setups can test against testnet

## Testing

- `make js`
- `/target/debug/frequency export-metadata --tmp ./js/api-augment/metadata.json && cd js/api-augment && npm run build`

### CODE REVIEW

File: node/cli/src/export_metadata_cmd.rs [hunk 1]
```diff
@@ -52,7 +52,7 @@ impl ExportMetadataCmd {
 		let maybe_hash = client.hash(block_number.into())?;
 		let block_hash = maybe_hash.ok_or_else(|| Error::from("Block not found"))?;
 
-		let metadata: Bytes = api.metadata(block_hash).unwrap().into();
+		let metadata: Bytes = api.metadata_at_version(block_hash, 15u32).unwrap().unwrap().into();
```
Comment (wilwade) [inline node/cli/src/export_metadata_cmd.rs:55]: metadata_at_version returns an option, thus the second unwrap after unwrapping the result

Comment (enddynayn) [reply node/cli/src/export_metadata_cmd.rs:55]: I’m not sure if I’m overlooking something, but it seems like metadata_at_version only accepts one argument. Could you please confirm?
https://github.com/frequency-chain/frequency/blob/9fd47cf90b23b3030749790aceffbc726bd3dd40/runtime/frequency/src/apis.rs#L88

**Review by enddynayn — APPROVED** [review]
👍 great!

File: node/cli/src/export_metadata_cmd.rs (hunk 1 above)
Comment (enddynayn) [reply node/cli/src/export_metadata_cmd.rs:55]: Ahh its wrapped around `impl_runtime_apis`.

**Review by saraswatpuneet — APPROVED** [review]

**Pull request merged**